tests/phpunit/includes/phpunit6/compat.php

```diff
@@ -14,6 +14,7 @@ class_alias( 'PHPUnit\Framework\AssertionFailedError', 'PHPUnit_Framework_Assert
 	class_alias( 'PHPUnit\Framework\TestSuite', 'PHPUnit_Framework_TestSuite' );
 	class_alias( 'PHPUnit\Framework\TestListener', 'PHPUnit_Framework_TestListener' );
 	class_alias( 'PHPUnit\Util\GlobalState', 'PHPUnit_Util_GlobalState' );
-	class_alias( 'PHPUnit\Util\Getopt', 'PHPUnit_Util_Getopt' );
-	class_alias( 'PHPUnit\Util\Test', 'PHPUnit_Util_Test' );
+	if ( class_exists( 'PHPUnit\Util\Getopt' ) ) {
+		class_alias( 'PHPUnit\Util\Getopt', 'PHPUnit_Util_Getopt' );
+	}
```

Comment:
What happens to the other alias `class_alias( 'PHPUnit\Util\Test', 'PHPUnit_Util_Test' );` ?

Reply:
Looking at the upstream `compat.php` file it seems that class may only be needed for testing specific tickets.
I am not sure we need it but I'm not certain is is unnecessary either so we better add it back in.